packages/vaadin-context-menu/src/vaadin-contextmenu-items-mixin.js

```diff
@@ -230,7 +231,15 @@ export const ItemsMixin = (superClass) =>
         } else if (component.localName === 'hr') {
           component.setAttribute('role', 'separator');
         }
-        this.theme && component.setAttribute('theme', this.theme);
+
+        const itemTheme = this.__stringOrArrayToString(item.theme);
```

Comment:
Let's align with the `vaadin-menu-bar` method implementation.

Reply:
Fixed